themes/bootstrap3/templates/search/advanced/solr.phtml

```diff
@@ -1,55 +1,51 @@
 <? if (!empty($this->facetList) || !empty($this->checkboxFacets)): ?>
-  <div class="row">
-    <fieldset class="col-sm-12">
-      <legend><?=$this->transEsc('Limit To')?></legend>
-      <? if (!empty($this->checkboxFacets)): ?>
-        <?=$this->render('search/advanced/checkbox-filters.phtml')?>
-      <? endif; ?>
-      <div class="row">
-        <? foreach ($this->facetList as $field => $list): ?>
-          <div class="col-sm-<?=floor(12/count($this->facetList)) ?>">
-            <label for="limit_<?=$this->escapeHtmlAttr(str_replace(' ', '', $field))?>"><?=$this->transEsc($list['label'])?>:</label>
-            <select class="form-control" id="limit_<?=$this->escapeHtmlAttr(str_replace(' ', '', $field))?>" name="filter[]" multiple="multiple" size="10">
-              <? if (is_array($this->hierarchicalFacets) && in_array($field, $this->hierarchicalFacets)): ?>
-                <? foreach ($list['list'] as $value): ?>
-                  <? $display = str_pad('', 4 * $value['level'] * 6, '&nbsp;', STR_PAD_LEFT) . $this->escapeHtml($value['displayText']); ?>
-                  <option value="<?=$this->escapeHtmlAttr(($value['operator'] == 'OR' ? '~' : '') . $field . ':"' . $value['value'] . '"')?>"<?=(isset($value['selected']) && $value['selected'])?' selected="selected"':''?>><?=$display?></option>
-                <? endforeach; ?>
-              <? else: ?>
-                <?
-                // Sort the current facet list alphabetically; we'll use this data
-                // along with the foreach below to display facet options in the
-                // correct order.
-                $sorted = array();
-                foreach ($list['list'] as $i => $value) {
-                  if (!empty($value['displayText'])) {
-                    $sorted[$i] = $value['displayText'];
-                  }
+  <fieldset class="solr-facets">
+    <legend><?=$this->transEsc('Limit To')?></legend>
+    <? if (!empty($this->checkboxFacets)): ?>
```

Comment:
I can also create a variation of the range problem on the Summon screen:
![image](https://user-images.githubusercontent.com/309069/26945837-2e9baadc-4c5b-11e7-96e1-54e12c07c05c.png)

As with Solr, this problem exists in master as well as this branch.